xunit-autolabeler-v2/ast_parser/python/source_parsers/direct_invocation.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any, List
+
+
+from ast_parser.python.drift_data_dict import make_drift_data_dict
+from ast_parser.lib.constants import IGNORED_METHOD_NAMES
+
+
+def parse(nodes: List[Any], class_name: str) -> List[Any]:
```

Comment:
Reminder to update this to List[DriftData] or whatever you name the namedtuple if you follow my previous suggestion.

Reply:
Done. (It's `ast.FunctionDef`.)

Reply:
**Edit:** reverted to `Any` so `mypy` passes - see above comment.